setup.md

```diff
@@ -0,0 +1,55 @@
+you will need to download the following Programs:
+IntelliJ
+Java 8 (You may need to uninstall Java 9)
+Java 8 SDK - You should choose the version that matches your Java
+git bash
+
+Run each of those, and wait for them to install.
+Next, open git bash and run the following:
+git clone you will need to download the following files:
```

Comment:
i think you have a copy past error or something here

Reply:
Yea I did have a copy/paste error. A few, in fact